members/us_sen_ga_kelly_loeffler.yaml

```diff
@@ -0,0 +1,131 @@
+bioguide: us_sen_ga_kelly_loeffler
+contact_form:
+  method: post
+  usePhantomJsLogging: true
+  steps:
+    - visit: "https://www.loeffler.senate.gov/"
+    - find:
+      - value: "#edit-prefix"
```

Comment:
I think this should be `selector` instead of `value`

Reply:
updated